angr/state_plugins/sim_action.py

```diff
@@ -120,6 +124,10 @@ def _desc(self):
     def all_objects(self):
         return [ a for a in ( self.target, self.condition ) if a is not None ]
 
+    @property
+    def is_symbolic(self):
+        return getattr(self.target, "symbolic", False)
```

Comment:
Why can't we do `return self.target.symbolic`?

Reply:
I used `getattr` because it also handle case when `self.target` doesn't have an attribute named `symbolic`. I ran some quick tests and found there were cases when the data members were `int` or some similar built-in data type rather than an AST. I was initially considering testing their type and then retrieving the attribute but this seemed better(but yes probably not very explicit)? If there's a better way this can be implemented, we could do it that way.

Reply:
I see. That's fine then.